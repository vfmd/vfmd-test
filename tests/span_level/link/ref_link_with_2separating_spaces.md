A [link]  [ref] to a url.

[ref]: http://example.net
