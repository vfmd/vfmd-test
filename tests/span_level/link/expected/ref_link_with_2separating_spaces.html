<p>A <a href="http://example.net">link</a> to a url.</p>
